src/main/java/de/qabel/desktop/daemon/sync/worker/index/sqlite/DesktopSyncDatabase.java

```diff
@@ -11,10 +11,22 @@ public DesktopSyncDatabase(Connection connection) {
         super(connection);
     }
 
+    private long version = 0;
```

Comment:
if you would need this long, then you could omit the default value because an uninitialized long has the value `0`